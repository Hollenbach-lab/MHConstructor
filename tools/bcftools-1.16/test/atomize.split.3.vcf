##fileformat=VCFv4.3
##contig=<ID=1,length=135006516>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	50	.	G	C,T	.	.	.	GT	2|1
