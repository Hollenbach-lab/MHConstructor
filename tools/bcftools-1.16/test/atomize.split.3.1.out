##fileformat=VCFv4.3
##FILTER=<ID=PASS,Description="All filters passed">
##contig=<ID=1,length=135006516>
##FORMAT=<ID=GT,Number=1,Type=String,Description="Genotype">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	sample
1	50	.	G	C	.	.	.	GT	.|1
1	50	.	G	T	.	.	.	GT	1|.
